common/app/model/DotcomContentType.scala

```diff
@@ -80,4 +81,8 @@ object DotcomContentType {
       case _ => None
     }
   }
+
+  def dotcomContentTypeFromPressedContent(storyContent: Option[PressedStory]): DotcomContentType = {
```

Comment:
you can call this method `apply` which would allow you to make it a constructor and call it like this `Pillar(storyContent)`

Reply:
Thanks for the tip, I've just made this change